src/rabbitmq.js

```diff
@@ -94,13 +101,31 @@ class RabbitMQ {
       )
     }
     this.tasks = opts.tasks || []
+    this.tasks = this.tasks.map(RabbitMQ._formatJobs)
     this.events = opts.events || []
+    this.events = this.events.map(RabbitMQ._formatJobs)
     this.log.trace({ opts: opts }, 'RabbitMQ constructor')
     joi.assert(this, optsSchema)
     this._setCleanState()
   }
 
   /**
+   * formates events and tasks to consistent format.
+   * add TID validation if not already there
+   * @param  {Object|String} item task/job item from map
+   * @return {Object}      formated job type
+   */
+  static _formatJobs (item: string|Object): Object {
```

Comment:
can you cover this function with unit test? Syntax for updating schema is a little bit strange so it would be nice to have uni test